lsp4clj/deps.edn

```diff
@@ -0,0 +1,10 @@
+{:deps {com.taoensso/timbre {:mvn/version "5.1.2"}
+        org.eclipse.xtend/org.eclipse.xtend.lib {:mvn/version "2.25.0"}
```

Comment:
timbre is already in common, right?
```suggestion
{:deps {org.eclipse.xtend/org.eclipse.xtend.lib {:mvn/version "2.25.0"}
```

Reply:
Yeah, but I think it is common to still depend on the stuff your libs depend on if you use it directly, right? I think we should treat common as "just a lib", right?

Reply:
Ok, sounds good